pkg/config/constants/constants.go

```diff
@@ -164,7 +164,9 @@ const (
 
 	RemoteGatewayClassName   = "istio-remote"
 	WaypointGatewayClassName = "istio-waypoint"
-	// TODO formalize this API
+
+	// GatewayNameLabel indicates the gateway managing a particular proxy instances. Only populated for Gateway API gateways
+	// TODO: Formalize this API
```

Comment:
close to merging https://github.com/kubernetes-sigs/gateway-api/pull/1757

Reply:
Nice! Will be great to get this standardized